### PR TITLE
Fix some mail templates

### PR DESCRIPTION
**Pull request by tsauerwein**
Some minor fixes in email templates.

### CODE REVIEW

File: de_DE.csv [hunk 1]
```diff
@@ -8933,7 +8934,7 @@ You may need more permissions to access this category.,Ihre Berechtigungen reich
 You may need more permissions to access this product.,Ihre Berechtigungen reichen nicht aus um diesen Artikel anzusehen.
 You moved the category,Sie haben die Kategorie verschoben.
 You must buy at least %1 of these per purchase.,Sie müssen mindestens %1 dieser Artikel pro Bestellung kaufen.
-You must confirm your %customer_email email before you can sign in (link is only valid once):,"Sie müssen Ihre %customer_email bestätigen, bevor Sie sich einloggen (Link ist nur einmal gültig):"
+You must confirm your %customer_email email before you can sign in (link is only valid once):,"Sie müssen Ihre %customer_email Email bestätigen, bevor Sie sich einloggen (Link ist nur einmal gültig):"
```
Comment (sydekumf) [inline de_DE.csv:8937]: What would be filled in %customer_email?

File: de_DE.csv [hunk 2]
```diff
@@ -3313,7 +3313,7 @@ If there is an account associated with %1 you will receive an email with a link
...
 "If you have any questions, please feel free to contact us at <a href=""mailto:%support_email"">%support_email</a> or by phone at %store_phone","Wenn Sie Fragen haben, kontaktieren Sie uns unter <a href=""mailto:%support_email"">%support_email</a>."
 "If you have not authorized this action, please contact us immediately at <a href=""mailto:%store_email"">%store_email</a>","Sollten Sie diese Änderung nicht autorisiert haben, kontaktieren Sie uns bitte umgehend unter <a href=""mailto:%store_email"">%store_email</a>"
-"If you have questions about your order, you can email us at <a href=""mailto:%store_email"">%store_email</a>","Wenn Sie Fragen zu Ihrer Bestellung haben, können Sie uns diese an <a href=""mailto:%store_email"">%store_email</a> schicken."
+"If you have questions about your order, you can email us at <a href=""mailto:%store_email"">%store_email</a>","Wenn Sie Fragen zu Ihrer Bestellung haben, können Sie uns diese an <a href=""mailto:%store_email"">%store_email</a> schicken"
```
Comment (sydekumf) [inline de_DE.csv:3316]: Why do you remove the dot? If you look up there are also dots.

Comment (tsauerwein) [reply de_DE.csv:3316]: There are two English translation strings "If you have questions about your order, ...", one ends with a dot, the other not. So far both German translations for these two strings had a dot at the end, which was wrong.

File: de_DE.csv (hunk 1 above)
Comment (tsauerwein) [reply de_DE.csv:8937]: The email address, for example "You must confirm your ab@c.de email before ...".

**Comment by sydekumf** [comment]
Thanks for the PR!

**Pull request merged**